docs/source/notes/autograd.rst

```diff
@@ -567,10 +567,10 @@ functions, and :math:`L` is real by our assumption that :math:`f` is a
 part of a real valued function, we have:
 
     .. math::
-        (\frac{\partial L}{\partial s})^* = \frac{\partial L}{\partial s^*}
+        \frac{\partial L}{\partial s} = (\frac{\partial L}{\partial s^*})^*
```

Comment:
This is equivalent to the previous equation, and I like better the symmetry of the previous equation, but fine.

Also, you may want to use `\left( ... \right)` to avoid getting tiny parens.

Reply:
yea it's equivalent, readers (like me) at this point has lost the reference of what grad_output is so I tried to match that, agree on the symmetry point tho

good point on \left(

Reply:
btw appreciate the pointer https://arxiv.org/pdf/2207.06114.pdf